2019-04-10 15:01:46.339 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-r_bTlc9dGwDV8G_YyR_lLw">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:01:46.340 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:38793 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-r_bTlc9dGwDV8G_YyR_lLw">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:01:46.340 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:38793 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:06:59.963 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-LbGy2p1bnI0_toRcedk77g">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:06:59.963 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:52491 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-LbGy2p1bnI0_toRcedk77g">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:06:59.964 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:52491 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:13:38.171 [error] <0.251.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.251.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-E_4jr841uYvAUyKWfxGMZw">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:13:38.171 [error] <0.248.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.248.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.250.0>, {<<"client 127.0.0.1:40103 -> 127.0.0.1:5672">>,1}) at <0.251.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-E_4jr841uYvAUyKWfxGMZw">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:13:38.171 [error] <0.248.0> Supervisor {<0.248.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.250.0>, {<<"client 127.0.0.1:40103 -> 127.0.0.1:5672">>,1}) at <0.251.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:21:44.706 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-HaBwDtWPCVbgZi_cLpPv8Q">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:21:44.706 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:44831 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-HaBwDtWPCVbgZi_cLpPv8Q">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:21:44.707 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:44831 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:23:20.090 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-chaB4AWhZ5whQSZmFcK0Kg">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:23:20.091 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:39357 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-chaB4AWhZ5whQSZmFcK0Kg">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:23:20.091 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:39357 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:23:39.564 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-032KSQS3I-5fElv_ARD7gQ">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:23:39.564 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:60107 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-032KSQS3I-5fElv_ARD7gQ">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:23:39.564 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:60107 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:25:44.066 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-qllutj32426daiQ91wfqew">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:25:44.066 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:33565 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-qllutj32426daiQ91wfqew">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:25:44.066 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:33565 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:38:10.238 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-Yx1z27O7bw7kB7G0ZOsTZg">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:38:10.239 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:46091 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-Yx1z27O7bw7kB7G0ZOsTZg">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:38:10.239 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:46091 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:41:59.180 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-I8gdxGbapcqUC8AY_JJT_Q">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:41:59.180 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:40287 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-I8gdxGbapcqUC8AY_JJT_Q">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:41:59.180 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:40287 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:50:48.092 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-AVh6Jie69ncuNVPnGXydfA">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:50:48.092 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:36773 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-AVh6Jie69ncuNVPnGXydfA">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:50:48.093 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:36773 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:52:58.894 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-eXDZ-xYjU3UVk7-XY0oqRw">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:52:58.895 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:37063 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-eXDZ-xYjU3UVk7-XY0oqRw">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:52:58.895 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:37063 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 15:53:38.247 [error] <0.250.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 CRASH REPORT Process <0.250.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-tUO_9j2tJJLXO42uV671rA">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185
2019-04-10 15:53:38.248 [error] <0.247.0>@rabbit_framing_amqp_0_9_1:shortstr_size:185 Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:39805 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason bad argument in call to erlang:size(#{consumer_count => 0,message_count => 0,queue => <<"amq.gen-tUO_9j2tJJLXO42uV671rA">>}) in rabbit_framing_amqp_0_9_1:shortstr_size/1 line 185 in context child_terminated
2019-04-10 15:53:38.248 [error] <0.247.0> Supervisor {<0.247.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.249.0>, {<<"client 127.0.0.1:39805 -> 127.0.0.1:5672">>,1}) at <0.250.0> exit with reason reached_max_restart_intensity in context shutdown
2019-04-10 20:15:19.296 [error] <0.235.0> gen_server 'Elixir.John' terminated with reason: {badkey,channel,#{chan => #{'__struct__' => 'Elixir.AMQP.Channel',conn => #{'__struct__' => 'Elixir.AMQP.Connection',pid => <0.238.0>},pid => <0.249.0>},conn => #{'__struct__' => 'Elixir.AMQP.Connection',pid => <0.238.0>},name => 'Elixir.John',queue => <<"amq.gen-CVxt4cGIXEo2O_-69SQ_yQ">>,tag => <<"amq.ctag-eVYvmNY-RnqvJnCc2qEKcg">>}} in 'Elixir.Hospital.Doctor':handle_cast/2 line 64
2019-04-10 20:15:19.318 [error] <0.235.0> CRASH REPORT Process 'Elixir.John' with 1 neighbours crashed with reason: {badkey,channel,#{chan => #{'__struct__' => 'Elixir.AMQP.Channel',conn => #{'__struct__' => 'Elixir.AMQP.Connection',pid => <0.238.0>},pid => <0.249.0>},conn => #{'__struct__' => 'Elixir.AMQP.Connection',pid => <0.238.0>},name => 'Elixir.John',queue => <<"amq.gen-CVxt4cGIXEo2O_-69SQ_yQ">>,tag => <<"amq.ctag-eVYvmNY-RnqvJnCc2qEKcg">>}} in 'Elixir.Hospital.Doctor':handle_cast/2 line 64
2019-04-10 21:03:07.233 [error] <0.235.0> CRASH REPORT Process <0.235.0> with 1 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'Elixir.Knee' in vhost '/'">>}},{gen_server,call,[<0.249.0>,{subscribe,{'basic.consume',0,<<"Elixir.Knee">>,<<>>,false,false,false,false,[]},<0.251.0>},60000]}} in gen_server:call/3 line 223
2019-04-10 22:29:25.204 [error] <0.235.0> gen_server l1 terminated with reason: {'module could not be loaded',[{knee,to_string,[],[]},{'Elixir.Hospital.Doctor',handle_cast,2,[{file,"lib/hospital/doctor.ex"},{line,64}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,637}]},{gen_server,handle_msg,6,[{file,"gen_server.erl"},{line,711}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
2019-04-10 22:29:25.222 [error] <0.235.0>@knee:to_string CRASH REPORT Process l1 with 1 neighbours crashed with reason: call to undefined function knee:to_string()
2019-04-10 22:30:31.130 [error] <0.235.0>@Elixir.Hospital.Doctor:handle_cast:64 gen_server l1 terminated with reason: bad argument in 'Elixir.Hospital.Doctor':handle_cast/2 line 64
2019-04-10 22:30:31.162 [error] <0.235.0>@Elixir.Hospital.Doctor:handle_cast:64 CRASH REPORT Process l1 with 1 neighbours crashed with reason: bad argument in 'Elixir.Hospital.Doctor':handle_cast/2 line 64
